README.md

```diff
@@ -5,15 +5,18 @@ This project contains an API server.
 
 ### Author: Marlene Rinker
 
-- [submission PR](https://github.com/marlenerinker-401-advanced-javascript/api-server/pull/9)
-- [tests report](https://github.com/marlenerinker-401-advanced-javascript/api-server/actions)
+- [Submission PR](https://github.com/marlenerinker-401-advanced-javascript/api-server/pull/11)
+- [Tests Report](https://github.com/marlenerinker-401-advanced-javascript/api-server/actions)
 - [Swagger Hub](https://app.swaggerhub.com/apis/marlene-rinker/api-server/0.1#/)
+- [Heroku deployment](https://app-server-mr401.herokuapp.com/)
 
 
 
 ### Setup
 
 #### `.env` requirements
+PORT=<port> (will default to 3000 if you don't specify here)
+
 MONGODB_ATLAS_URI=mongodb+srv://dbAdmin:<password>@cluster0-2u75g.mongodb.net/<database>?retryWrites=true&w=majority
```

Comment:
You may want to put this on as `MONGODB_ATLAS_URI=<path to mongo>` or something similar. Not everyone will have Atlas downloaded, my URI is different than this.